scripts/modal.js

```diff
@@ -242,17 +302,26 @@ class Modal extends View
                               <p>Name: <input type="text" id="name" value="${this.obj.getName()}"></p> 
                               <p>Description: <input type="text" id="nodeDescriptionBox" value="${this.obj.functionDescription}"> </p>
                               <p id="functionDefinition"> Function: ${this.obj.functionDefinitionInstance ? this.obj.functionDefinitionInstance.name : "No function assigned"} </br> </p>
+                              <div id="functionVersion"> Version: 
+                                <button id="functionVersionDown" class="btn"></button> 
+                                <span id="versionNumber"> 0 </span> 
+                                <button id="functionVersionUp" class="btn"></button>
+                              </div>
                               Variables:
                               <ul id="cVarList"></ul>
                             </div>`);
-      
+
+
+      this.lowerVersion  = new EarlierVersionButton();
```

Comment:
Abstrahera?, Det kan va bra att inte låta dem sitta löst i modal, typ att man tänker  att Modal är modal och versionshantering fliken är versionshanteringsfliken som är en komponent i Modal som har buttons i sig och får in en array med alla versioner som den kan visa. Skriver denna för vidare tänka på det, kan va en task i sig. 
Ursäkta att jag tjatar om sånt, men för vår skull så att vi inte glömmer.